### PR TITLE
feat(Themes): merge theme and page template settings

### PR DESCRIPTION
**Pull request by fabriciomendonca**
This PR updates the `usePageTemplateSettings` hook to return merged theme and page template settings.
Also some refactoring to reduce the complexity of the previous implementation.

### CODE REVIEW

File: packages/app-bridge/src/react/usePageTemplateSettings.spec.ts [hunk 1]
```diff
@@ -1,19 +1,41 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { cleanup, renderHook, waitFor } from '@testing-library/react';
-import { afterEach, beforeEach, describe, expect, it, vi } from 'vitest';
+import { Mock, afterEach, beforeEach, describe, expect, it, vi } from 'vitest';
 import { getAppBridgeThemeStub } from '../tests';
 import { usePageTemplateSettings } from './usePageTemplateSettings';
+import { useThemeSettings } from '.';
+// import { useThemeSettings } from './useThemeSettings';
```
Comment (SamuelAlev) [inline packages/app-bridge/src/react/usePageTemplateSettings.spec.ts:8]:
```suggestion
import { useThemeSettings } from './useThemeSettings';
```

Comment (fabriciomendonca) [reply packages/app-bridge/src/react/usePageTemplateSettings.spec.ts]: Done. Thanks for the catch

File: packages/app-bridge/src/react/usePageTemplateSettings.spec.ts [hunk 2]
```diff
@@ -111,4 +136,92 @@ describe('usePageTemplateSettings', () => {
 
         expect(console.error).toHaveBeenCalledOnce();
     });
+
+    describe('Theme and Page template overrides', () => {
+        beforeEach(() => {
+            (useThemeSettings as Mock).mockImplementation(() => ({ themeSettings: THEME_SETTINGS }));
+        });
+        it('returns the page template settings merged with theme settings and the list of custom overrides', async () => {
```
[the remark this answers is not in the document]
Comment (fabriciomendonca) [reply packages/app-bridge/src/react/usePageTemplateSettings.spec.ts]: Done

**Review by mike85 — APPROVED** [review]
Looks good to me!

**Pull request merged**